Android.common.mk

```diff
@@ -137,7 +137,7 @@ LOCAL_CPPFLAGS += \
 	-DENABLE_RBC
 endif
 
-ifneq ($(strip $(HWC_DISABLE_VA_DRIVER)), true)
```

Comment:
No nessary change make file. On GP2.0 it DISABLE_VA_DRIVER is not defined.
Discard this change